setup/configuration/index.md

```diff
@@ -62,7 +66,33 @@ spring:
           token: [vault access token]
 ```
 
-For information about configuring other supported Config Server backends, see the [Spring Cloud Config Server documentation](https://cloud.spring.io/spring-cloud-static/spring-cloud-config/2.1.0.RELEASE/single/spring-cloud-config.html#_environment_repository).
+> The above example configures authentication for Vault using the Vault
+> `X-Config-Token` header, via the setting of the
+> `spring.cloud.config.server.vault.token` property. In Spinnaker 1.18, you  can
```

Comment:
```suggestion
> `spring.cloud.config.server.vault.token` property. Starting with Spinnaker 1.18, you  can
```

Reply:
replace
`using the Vault X-Config-Token header, via the setting ...`
with
`using a static authentication token, via the setting ...`